### ISSUE TITLE
Not all file uploads are removed on submission deletion

### ISSUE THREAD

**Issue by roland-d** [verbb/formie#1134]
### Describe the bug

When you have a form with 2 or more FileUpload fields and the option is set to `Delete Files` when a submission is deleted, only the last FileUpload field asset is deleted from the server.

### Steps to reproduce

1. Create a form with 2 FileUpload Fields
2. Set the option `File Uploads` on the `Settings` tab to `Delete Files`
3. Submit the form
4. Check that you have 2 uploaded files on disk in the location of the volume
5. Delete the submission
6. Check that only 1 file has been deleted

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

Craft Pro 4.2.5.2

### Plugin version

2.0.12

### Multi-site?

No

### Additional context

So I traced this back to the file `src/elements/Submission.php` on line 1088 where the following code is executed:
```
$this->_assetsToDelete = $this->getFieldValue($field->handle)->all();
```
Since the `_assetsToDelete` is an array the content is always overwritten by the next FileUpload. This is also why only 1 file is deleted.

Changing this line to the following, fixes the issue for me:
```
$this->_assetsToDelete = array_merge(($this->_assetsToDelete ?? []), $this->getFieldValue($field->handle)->all());
```

**Comment by engram-design** [comment]
I feel like that was due to a bad merge at some point, because as you say that makes zero sense! Should be fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
